src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentsService.kt

```diff
@@ -73,8 +75,12 @@ class AppointmentsService(
     additionalInformation: String?
   ): ActionPlanAppointment {
     val appointment = getActionPlanAppointmentOrThrowException(actionPlanId, sessionNumber)
+
+    if (appointment.sessionFeedbackSubmittedAt != null) {
+      throw ResponseStatusException(HttpStatus.UNPROCESSABLE_ENTITY, "session feedback has already been submitted for this appointment")
```

Comment:
Is this not a 409 Conflict

Reply:
Looks like 409 Conflict works for this use case 👍 

https://httpstatuses.com/409

> The request could not be completed due to a conflict with the current state of the target resource. This code is used in situations where the user might be able to resolve the conflict and resubmit the request.